lgi/lgi.h

```diff
@@ -29,6 +29,29 @@
 #endif
 #endif
 
+/* Lua 5.3 integers. */
+#if LUA_VERSION_NUM >= 503
+// For Lua 5.3
+typedef LUA_UNSIGNED lgi_Unsigned;
+#if LUA_INT_TYPE == LUA_INT_INT
+#define LGI_LUAINT_FORMAT "%d"
+#elseif LUA_INT_TYPE == LUA_INT_LONG
+#define LGI_LUAINT_FORMAT "%ld"
+#else
+#define LGI_LUAINT_FORMAT "%lld"
+#endif
+#elif LUA_VERSION_NUM == 502
+// For Lua 5.2
+typedef lua_Unsigned lgi_Unsigned;
+#define lua_isinteger(L, i) 0
+#define LGI_LUAINT_FORMAT "%td"
```

Comment:
I had to check the man page of `printf`: Why do you think that this is `ptrdiff_t` argument?